samples/awssqs_source/README.md

```diff
@@ -0,0 +1,95 @@
+# AWS SQS - Source
+
+## Deployment Steps
+
+### Prerequisites
+
+1. Create an [AWS SQS queue](https://aws.amazon.com/sqs/).
+
+1. Setup [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
+1. Install the [in-memory `ClusterChannelProvisioner`](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel).
+    - Note that you can skip this if you choose to use a different type of `Channel`. If so, you will need to modify `channel.yaml` before deploying it.
+1. Create a `Channel`. You can use your own `Channel` or use the provided sample, which creates `qux-1`. If you use your own `Channel` with a different name, then you will need to alter other commands later.
+
+    ```shell
+    kubectl -n default apply -f samples/awssqs-source/channel.yaml
+    ```
+
+1. Acquire [AWS
+   Credentials](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html)
+   for the same account.  Your credentials file should look like this:
+
+       [default]
+       aws_access_key_id = ...
+       aws_secret_access_key = ...
+
+        1. Create a secret for the downloaded key:
+
+            ```shell
+            kubectl -n knative-sources create secret generic awssqs-source-credentials --from-file=credentials=PATH_TO_CREDENTIALS_FILE
+            ```
+
+1. Create an AWS SQS queue. Replace `QUEUE_URL` with your desired Queue URL.
+
+### Deployment
+
+1. Deploy the `AwsSqsSource` controller as part of eventing-source's controller.
+
+    ```shell
+    ko -n default apply -f config/default-awssqs.yaml
+    ```
+
+    - Note that if the `Source` Service Account secret is in a non-default location, you will need to update the YAML first.
+
+1. Replace the place holders in `awssqs-source.yaml`.
+    - `QUEUE_URL` should be replaced with your AWS SQS queue URL.
+    - `qux-1` should be replaced with the name of the `Channel` you want
+      messages sent to. If you deployed an unaltered `channel.yaml`
+      then you can leave it as `qux-1`.
+    - `awsCredsSecret` should be replaced if you are using a non-default
+      secret or key name for the receive adapter's credentials.
+
+1. Deploy `awssqs-source.yaml`.
```

Comment:
I've recently started giving the command using sed for the replacements, which I think makes it easier. So any early step would be `export QUEUE_URL=REPLACE_ME` and `export AWS_REGION=REPLACE_ME`, then this step is

```shell
sed "s!QUEUE_URL!$QUEUE_URL!g" samples/awssqs-source/awssqs-source.yaml | sed "s/AWS_REGION/$AWS_REGION/g" | kubectl -n default apply -f -
```